notebook/static/lab/plugin.js

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Jupyter Development Team.
```

Comment:
This file and its css will be moved to an `application` plugin in `jupyter-js-plugins` soon, pending some updates to `phosphide`